_includes/br/What-is.md

```diff
@@ -1,36 +1,34 @@
 [Julia](https://julialang.org) é uma linguagem de programação de código aberto,
 multi-plataforma, de alto nível e alta performance para computação técnica.
 
-Julia has an
-<a class="tooltip" href="#">LLVM<span> Low-Level Virtual Machine (LLVM) is a compiler
-infrastructure to build intermediate and/or binary machine code.
+Julia tem um
+<a class="tooltip" href="#">LLVM<span> Máquina Virtual de Baixo Nível (LLVM) é um compilador
+infra-estrutura para construir código de máquina intermediário e/ou binário.
 </span></a>-based
-<a class="tooltip" href="#">JIT<span> Just-In-Time compilation occurs at
-run-time rather than prior to execution, which means it offers both the
-speed of compiled code and the flexibility of interpretation. The
-compiler parses the code and infers types, after which the LLVM code is
-generated, which in turn is compiled into native code. </span></a>
-compiler that allows it to match the performance of languages such as C
-and FORTRAN without the hassle of low-level code. Because the code is
-compiled on the fly you can run (bits of) code in a shell or
-<a class="tooltip" href="#">REPL <span> Read-Eval-Print-Loop </span></a>,
-which is part of the recommended
-[workflow](https://docs.julialang.org/en/stable/manual/workflow-tips) .
+<a class="tooltip" href="#">JIT<span> Compilação 'just-in-time' ocorre em
```

Comment:
Difícil de escrever esta.

Julia tem um compilador
JIT
baseado em
LLVM

Só que no JIT fica
`<a class="tooltip" href="#">JIT<span> Compilação 'just-in-time' ocorre em...`
e no LLVM
`<a class="tooltip" href="#">LLVM<span> Máquina Virtual de Baixo Nível (LLVM) é um compilador`